tests/unit/test_charm.py

```diff
@@ -11,13 +11,26 @@
 
 from charm import TLSRequirerOperatorCharm
 
+PRIVATE_KEY = "whatever private key"
+PRIVATE_KEY_PASSWORD = "whatever password"
+CSR = "whatever csr"
```

Comment:
I still see a couple of places in the test that use "whatever csr" instead of referencing this variable

Reply:
Done